airflow/metrics/validators.py

```diff
@@ -88,25 +88,13 @@ class MetricNameLengthExemptionWarning(Warning):
 
 def get_validator() -> ListValidator:
     validators = {
-        "basic": {"allow": AllowListValidator, "block": BlockListValidator},
         "pattern": {"allow": PatternAllowListValidator, "block": PatternBlockListValidator},
```

Comment:
```suggestion
       "allow": PatternAllowListValidator, 
       "block": PatternBlockListValidator,
```

I don't see a reason to keep the "pattern" level anymore.